.env.sample

```diff
@@ -18,3 +18,5 @@ DJANGO_SETTINGS_MODULE=
 NODE_ENV=
 DEBUG=
 ENVIRONMENT=
+REACT_APP_ACCESS_TOKEN="551b10b0c4164b58f566889e0a6b1880aa4680ae"
+REACT_APP_API_URI="https://localhost:8000/graphql"
```

Comment:
Add a trailing line here. You can actually configure your editor to automatically add this for you